diaries.html

```diff
@@ -1435,6 +515,19 @@ <h4>Memories</h4>
     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jQuery.Marquee/1.5.0/jquery.marquee.js"></script>
     <script type="text/javascript" src="js/lity.js"></script>
     <script type="text/javascript" src="//buttons.github.io/buttons.js" async defer></script>
+    <script type="text/javascript" src="js/data.js"></script>
     <script type="text/javascript" src="js/custom_script.js"></script>
+    <script>
+        let filterUl = document.getElementById("filter-ul");
+        let buttonLi = filterUl.getElementsByClassName("btn");
+
+        for (let i = 0; i < buttonLi.length; i++) {
+            buttonLi[i].addEventListener("click", function() {
+            let current = document.getElementsByClassName("active-filter-button");
+            current[0].className = current[0].className.replace(" active-filter-button", "");
+            this.className += " active-filter-button";
+        });
+        }

```

Comment:
code is not needed. I think it's bad to call 2 listeners for the click-on button.